users/service.js

```diff
@@ -0,0 +1,66 @@
+var response = require('../Objects/response.js');
+var userModels = require('./userModels');
+var Student = userModels.Student;
+var Admin = userModels.Admin;
+
+var createUser = (json) => {
+  if (json.role == 'Student') {
+    // Create new student.
+    var student = new Student({
+      approvalStatus: false,
+      email: json.email,
+      firstName: json.firstName,
+      lastName: json.lastName,
+      points: 0,
+      role: 'Student',
+      studentId: getUid();
```

Comment:
It's a UID, I wasn't sure how UID generation would work. For now I'll just change it to 0.